packages/terra-abstract-modal/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed title getting announced twice.
```

Comment:
Do we not want to mention about addition of two new aria props?

Reply:
Abstract modal is deprecated. So its ok since consumers will not be using this directly instead they should be using modal mamanger.